go/vt/wrangler/testlib/backup_test.go

```diff
@@ -46,7 +45,31 @@ import (
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 )
 
+type compressionDetails struct {
+	BuiltinCompressor       string
+	BuiltinDecompressor     string
+	ExternalCompressorCmd   string
+	ExternalCompressorExt   string
+	ExternalDecompressorCmd string
```

Comment:
It seems like the only reason all of these flags have to be exported from the mysqlctl package is so that we can use them for testing here.
Can you look into whether we can move this whole backup_test into the mysqlctl package? Then we don't have to expire the flags.